TodoAppWithSpringboot/src/main/java/com/webonise/exception/EmptyTodoListFoundException.java

```diff
@@ -0,0 +1,5 @@
+package com.webonise.exception;
+
+public class EmptyTodoListFoundException extends RuntimeException{
+	private static final long serialVersionUID = 1L;
```

Comment:
nice, for more improvising yourself check "global exception handling" https://dzone.com/articles/global-exception-handling-with-controlleradvice